.github/workflows/forms-flow-web-cd.yml

```diff
@@ -1,92 +1,34 @@
-name: Push Forms flow Web to registry
-
+name: forms-flow-web-CD
```

Comment:
Wouldn't this replace the CD action where we build and push the image ?

Reply:
yes, since we do not need the image anymore

Reply:
we only need the image for the root config.